test-resources/headless-core-stub/lambdas/start/src/errors/error-response.ts

```diff
@@ -0,0 +1,21 @@
+import { Logger } from "@aws-lambda-powertools/logger";
+import { HeadlessCoreStubError } from "./headless-core-stub-error";
+
+export const handleErrorResponse = (err: unknown, logger: Logger) => {
+    if (err instanceof Error) {
+        logger.error(err.message, err);
```

Comment:
Do you think there might be something helpful that we can log even if the `err` isn't an `Error`?